src/cloudproof_py/anonymization/anonymize.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+import json
+from datetime import timezone
+from typing import Callable, Dict, Optional
+
+import pandas as pd
+from cloudproof_anonymization import (
+    DateAggregator,
+    Hasher,
+    NoiseGenerator,
+    NumberAggregator,
+    NumberScaler,
+    WordMasker,
+    WordPatternMasker,
+    WordTokenizer,
+)
+from cloudproof_fpe import Alphabet, Integer
+from dateutil import parser as date_parser
+from humps import decamelize
+
+DURATION_IN_SECONDS = {
+    "Second": 1,
+    "Minute": 60,
+    "Hour": 3600,
+    "Day": 86400,
```

Comment:
no week?

Reply:
good question, we though about that with Celia but decided not to add it